test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -178,6 +222,8 @@ internal CommandRunnerResult PackSolution(string workingDirectory, string soluti
 
         private CommandRunnerResult PackProjectOrSolution(string workingDirectory, string file, string args, string nuspecOutputPath, bool validateSuccess)
         {
+            CreateTempGlobalJson(workingDirectory);
```

Comment:
I don't think it needs added here. 
It should be done at creation time.

Reply:
Thanks! Removed already. Only keep CreateTempGlobalJson in CreateDotnetToolProject  and CreateDotnetNewProject. Will check if manually fix for other tests afterwards.